client/policy_validator.go

```diff
@@ -348,9 +350,22 @@ func (p *PolicyAPIClient) execGet(args ...string) (*PolicyAPIResult, error) {
 	fullArgs := []string{"get", "policies"}
 	fullArgs = append(fullArgs, args...)
 	fullArgs = append(fullArgs, "-o", "json")
-	out, err := p.cli.exec(fullArgs, p.cli.GetNamespace())
+	var out *bytes.Buffer
+	err = utils.RetryWithContext(ctx, time.Millisecond*100, func() (bool, error) {
```

Comment:
Is there a reason to retry every tenth of second?  I imagine people might put rate limiting on their clusters, and I guess this would generate some 20 requests per second.

Reply:
I think it can be increased to 1s.

Reply:
Actually 1s does not feel right, as it is being done twice here. So I will use 200ms which is a value that still feels
ok from a user experience standpoint.